### PR TITLE
feat: Created User model with associations.

### PR DESCRIPTION
**Pull request by zcaceres**

### CODE REVIEW

File: db/models/user.js [hunk 1]
```diff
@@ -2,18 +2,32 @@
...
 module.exports = db => db.define('users', {
-  name: STRING,
+  name: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      isAlphanumeric: true,
+    }
+  },
   email: {
```
Comment (abc516) [inline db/models/user.js]: should also put allowNull property and set it to 'false'

Comment (abc516) [inline db/models/user.js:15]: should have allowNull property set to false

Comment (zcaceres) [reply db/models/user.js:15]: took care of it! please check most recent commit

**Review by abc516 — COMMENTED** [review]
Only need to add allowNull property to email field. Otherwise, looks great!

**Review by abc516 — APPROVED** [review]

**Pull request merged**